src/types/positions.ts

```diff
@@ -11,6 +12,7 @@ export interface PositionsHook {
   getPositionDefinitions(
     networkId: NetworkId,
     address?: string,
+    t?: TFunction<'translation', undefined>,
```

Comment:
Wanted to make this required, but there are a lot of tests that use it, wasn't sure how to mock this function for a test (typescript was yelling at me when using jest.fn()), and there were a lot of places it would have to be added.

Reply:
I propose we follow what I did in https://github.com/valora-inc/hooks/pull/505
and apply it here too.